environment-dev.yml

```
@@ -54,3 +54,5 @@ dependencies:
       - flake8
       # The `numpydoc` version should match the version installed in the `lint-pydocstyle` job of the CI.
       - numpydoc==1.1.0
+      # experimental version of fuzzydata requires atleast 0.0.4 to succesfully resolve all dependencies
+      - fuzzydata>=0.0.5
```

```suggestion
      # experimental version of fuzzydata requires at least 0.0.6 to successfully resolve all dependencies
      - fuzzydata>=0.0.6
```
Should we change this to >=0.0.6? As I see, you released new patches that are probably required to be in place for correct Modin testing.